index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <link rel="stylesheet" type="text/css" href="styles.css">
   </head>
-  <body>
+  <section>
     <header>
```

Comment:
every HTML doc needs a `<body>` so I wouldn't recommend removing that. instead i'd replace the `<div>` with `<section>`